modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-an-existing-ingress-controller_{context}"]
+= Modifying an existing Ingress Controller
+
+As a cluster administrator, you can modify an existing Ingress Controller to manage DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* DNS record created.
+
+.Procedure
+
+. Modify an `IngressController` custom resource to set `dnsManagementPolicy`:
+
++
+[source,terminal]
+----
+oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"loadBalancer":{"dnsManagementPolicy":"Unmanaged"}}}}'
+----
+
++
+[NOTE]
+====
+Once you update `.spec.endpointPublishingStrategy.loadBalancer.dnsManagementPolicy` field to `Unmanaged`, the existing DNS record must denote the same using the `dnsManagementPolicy` field. You can either retain or delete the DNS record in the cloud provider.
```

Comment:
This part I think we don't need to mention, sounds like it confuses the user. I've removed this in the suggestion added below.

Reply:
done